app/src/main/java/com/esoxjem/movieguide/listing/MoviesListingFragment.java

```diff
@@ -171,6 +171,13 @@ public void onSaveInstanceState(Bundle outState) {
         outState.putParcelableArrayList(Constants.MOVIE, (ArrayList<? extends Parcelable>) movies);
     }
 
+    public void searchViewCliked(String searchText){
```

Comment:
just a typo `Clicked` instead of `Cliked`

Reply:
corrected